### DISCUSSION TITLE
Custom Camera Source displays only white.

### DISCUSSION THREAD

**Discussion by Whoops-all-chaingunners** [OrcaSlicer/OrcaSlicer#4937] [Q&A]
Ip camera setup which is working fine in any browser only displays white in orca slicer when IP is set in Custom Camera Source. Can not find much of any mention of this feature anywhere.

**Comment by pickanothername** [comment]
I also have this issue in the current version of OrcaSlicer (2.1.1).
If someone who knows can confirm if this is the correct syntax, that would be much appreciated:
`rtsp://username:password@192.168.x.xxx/live`

If this is the correct syntax, then this is most likely an OrcaSlicer bug, since my stream also works well in VLC, etc.
Thanks!
